src/Indigo/Contracts/Holdings/Types.hs

```diff
@@ -107,8 +107,8 @@ instance TypeHasDoc (ML.StorageSkeleton StorageFields) where
     "Storage used in ManagedLedger-like contracts. \
     \The main field of this skeletion is `ledger`. It is a `big_map` that \
     \maps each registered address to its balance and approvals amounts. \
-    \Apart from `ledger` is stores `fields` that can vary for different contracts. \
-    \See `StorageFields` types."
+    \Apart from `ledger` is stores `fields` that can vary for different contracts \
+    \and fixed for this specific contract. See `StorageFields` type."
```

Comment:
Shouldn't it be "is fixed" btw? Or "are fixed" because `fields` is plural :thinking: